docs/management/dashboard_only_mode/index.asciidoc

```diff
@@ -2,25 +2,30 @@
 [[xpack-dashboard-only-mode]]
 == Kibana Dashboard Only Mode
 
-{xpack} gives the ability for admins to limit what certain users see when they log in to Kibana with a
-reserved `kibana_dashboard_only_user` role.
+{xpack} gives admins the ability to assign users a reserved `kibana_dashboard_only_user` 
+role that limits what they see when they log in to {kib}.
```

Comment:
I'm actually not sure what "reserved" role means in this case. I think it would be better to refer to it as a "built-in role", as in https://www.elastic.co/guide/en/elastic-stack-overview/6.3/built-in-roles.html.  So maybe something like this:

"If {security} is enabled, you can use the `kibana_dashboard_only_user` built-in role to limit ...."